docs/javascript/v2/changelog/release-notes.mdx

```diff
@@ -15,6 +15,18 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
 
+## 2023-09-13
+Released: `@100mslive/hms-video-store@0.10.15`, `@100mslive/react-sdk@0.8.15`, `@100mslive/hls-player@0.1.15`, `@100mslive/roomkit-react@0.1.6`
+
+### Important
+
+Prebuilt is now updated. Now supports changing themes and behaviour using dashboard.
+Details [here](https://www.100ms.live/docs/javascript/v2/quickstart/prebuilt-quickstart)
+
+### Fixed: 
+- Capture video at correct aspect ratio for mobile web - `@100mslive/hms-video-web`
+- Screenshare track was showing presenter video track - `@100mslive/hms-video-web`
```

Comment:
this was on role change right?

Reply:
no, We we stop screenshare, presenter video is shown